File: sdkwork-app-sdk-csharp/Models/ContactGroupVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class ContactGroupVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? Id { get; set; }
        public string? Name { get; set; }
        public List<string>? MemberIds { get; set; }
    }
}

File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/ContactGroupVO.java
package com.sdkwork.app.model;

public class ContactGroupVO {
    private String createdAt;
    private String updatedAt;
    private String id;
    private String name;
    private List<String> memberIds;

    public String getCreatedAt() {
        return this.createdAt;
    }
    
    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public String getUpdatedAt() {
        return this.updatedAt;
    }
    
    public void setUpdatedAt(String updatedAt) {
        this.updatedAt = updatedAt;
    }

    public String getId() {
        return this.id;
    }
    
    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return this.name;
    }
    
    public void setName(String name) {
        this.name = name;
    }

    public List<String> getMemberIds() {
        return this.memberIds;
    }
    
    public void setMemberIds(List<String> memberIds) {
        this.memberIds = memberIds;
    }
}
